tweets-service/service.yml

```diff
@@ -25,14 +25,25 @@ messages:
     - tweet deleted
     - tweet not created
 
-
-dependencies:
+dependencies: # comment out this dependency for deployment
   - name: mongo
     version: 3.4.0
 
-
-# production:
-#   aws:
-#     cpu: 100
-#     memory: 500
-#     essential: true
+environment:
+  default:
+  development:
+  production:
+    ROLE: exosphere-tweets-service
+    EXOCOM_PORT: 80
+    EXOCOM_HOST: exocom.space-tweet.local
```

Comment:
@hugobho I'd like to be able to remove these. These env variables are auto-added during the dev env and I think they are something we should be auto-adding in production as well. Do you think thats possible? If so, I'd like to create a ticket for this and make it a priority.